test/test_api/test_interpreter.py

```diff
@@ -440,5 +440,10 @@ def syslogs_to_df():
     # Apparently the function starts on the line where the decorator starts.
     assert c.line == syslogs_to_df.__wrapped__.__code__.co_firstlineno + 1
 
-def test_infer_from_stub():
-    assert True
+def test_core_module_completes():
+    import sys
+    import importlib
+    for module_name in sys.builtin_module_names:
```

Comment:
While I endorse your efforts, I feel like this is not necessary. This just takes longer than we need. Either test just `sys` or test a couple of modules like `time`, `sys` and maybe `gc`. You could do that easily with parametrize:

```
@pytest.mark.parametrize('module_name', ['sys', 'time'])
def test_core_module_completes(module_name):
```

It's a nice way to create multiple tests. BTW, even if we were to test all builtin modules, you should probably do it like this: `pytest.mark.parametrize('module_name', sys.builtin_module_names)`.